projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx

```diff
@@ -56,7 +82,11 @@ export default function VideoPressEdit( {
 					<DetailsPanel { ...{ attributes, setAttributes } } />
 				</InspectorControls>
 			) }
-			<View style={ style[ 'wp-block-jetpack-videopress__video-player' ] } />
+			<Player
+				html={ preview.html }
```

Comment:
The web block uses caching before passing the html to the player. 
That caching seems to be used when the video is being replaced. 
We might need to add in caching but for now I prefer to keep this PR light and follow up on that later if needed.